app/helpers/application_helper.rb

```diff
@@ -3,10 +3,41 @@
 #
 #  = Application Helpers
 #
-#  These methods are available to all templates in the application:
+#  Methods available to all templates in the application:
```

Comment:
Documents a list of methods for convenience. I've found that handier than looking through the whole file.